src/components/YourStocks/StocksList.jsx

```diff
@@ -70,7 +70,6 @@ function StocksList({
   const handleSharesSubmit = ((e) => {
     setSharesInput(e.target.value);
   });
-  console.log('BANK BALANCE', bankBalance);
   return (
     <>
```

Comment:
don't use empty tags. Either add div to the tag or just remove them entirely

Reply:
Tables cannot appear as a child of <div> and you must have a parent element for .jsx. So the only option to not get an error is to have <React.Fragment>.